pkg/apiserver/webhook/cloudentityselector_webhook.go

```diff
@@ -236,6 +195,14 @@ func (v *CESValidator) validateUpdate(req admission.Request) admission.Response
 			"%s (old:%v, new:%v)", errorMsgAccountNameUpdate, oldAccName, newAccountName))
 	}
 
+	// account namespace update not allowed.
```

Comment:
Why are we not allowing this? Any specific reason?
```suggestion
	// account Namespace update not allowed.
```

Reply:
Right now we do not support account name update. I didn't know the reason for this, however when I thought about it, I thought the reason might be due to the account config, account config is tied up to account name and namespace, we might not want to update it. Hence, added a same check for namespace too.
Let's discuss this.

Reply:
I did some research on this, Lets say CPA1 and CES1 are added. Also, CPA2 is added. If we update referenced account from CPA1 to CPA2 in CES1, Account config specific to CPA1 will continue to filters/other configs from CES1 even though its no more attached to CES1(Also, account poller would be fetching vm resources from cloud for CPA1). Cleanup needs a special handling. 
So, this is the reason why account name and and account namespace update should be blocked I feel. 
If we want to allow, we have to handle above cases.

Reply:
nit: account Namespace update not allowed.